tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -783,6 +783,26 @@ public async Task AuthorizationCodeRequestTestAsync()
                 }).ConfigureAwait(false);
         }
 
+        [TestMethod]
```

Comment:
We can have a pretty good **headless** E2E test here (can be done in a different commit). 

1. Acquire token in a headless way, for example U/P
2. Get the RT from the cache (this will be an interal call, users can't do this)
3. Clear the cache
4. AcquireTokenByRefreshToken
5. Asserts

Reply:
sounds good, Ill see if I can work something out.

Reply:
is this in progress?

---
In reply to: [246234142](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/759#discussion_r246234142) [](ancestors = 246234142)